### PR TITLE
8341138: Rename jtreg property docker.support as container.support

### PR DESCRIPTION
**Pull request by rsunderbabu**
The System property "docker.support" defined in VMProps gives a wrong impression that it is tied to docker alone. The property is common for any container runtime. Hence, it needs to be renamed as "container.support".

Positive Testing:
tier1,tier2,tier3 - to check stability
tier5 - to run container tests

Negative Testing:
Ran the following groups in hosts where container is not present.
open/test/hotspot/jtreg/containers/
open/test/jdk/jdk/internal/platform/docker/
open/test/jdk/jdk/internal/platform/cgroup/

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8341138](https://bugs.openjdk.org/browse/JDK-8341138): Rename jtreg property docker.support as container.support (**Enhancement** - P4)

### Reviewers
 * [Severin Gehwolf](https://openjdk.org/census#sgehwolf) (@jerboaa - **Reviewer**)
 * [Mikhailo Seledtsov](https://openjdk.org/census#mseledtsov) (@mseledts - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21423/head:pull/21423` \
`$ git checkout pull/21423`

Update a local copy of the PR: \
`$ git checkout pull/21423` \
`$ git pull https://git.openjdk.org/jdk.git pull/21423/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21423`

View PR using the GUI difftool: \
`$ git pr show -t 21423`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21423.diff">https://git.openjdk.org/jdk/pull/21423.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21423#issuecomment-2402589048)

### CODE REVIEW

File: test/lib/jdk/test/lib/Container.java [hunk 1]
```diff
@@ -23,9 +24,9 @@
 package jdk.test.lib;
 
 public class Container {
-    // Use this property to specify docker location on your system.
+    // Use this property to specify container location on your system.
```
Comment (jerboaa) [inline test/lib/jdk/test/lib/Container.java:27]:
```suggestion
    // Use this property to specify container runtime location (e.g. docker) on your system.
```

**Review by jerboaa — COMMENTED** [review]
Seems OK to me. One minor nit.

**Review by jerboaa — APPROVED** [review]

**Comment by mseledts** [comment]
Changes look good to me. Thank you for making these changes.

**Review by mseledts — APPROVED** [review]

**Pull request closed without merging**